web_pivot_computed_measure/static/src/pivot/pivot_model.esm.js

```diff
@@ -216,20 +218,31 @@ patch(PivotModel.prototype, {
                 const afield = toAnalyze.shift();
                 const fieldDef = this.metaData.fields[afield];
                 // Need to check if fieldDef exists to avoid problems with __count
-                if (fieldDef && fieldDef.__computed_id) {
-                    const cm = this._computed_measures.find((item) => {
-                        return item.id === fieldDef.__computed_id;
-                    });
-                    toAnalyze.push(cm.field1, cm.field2);
-                    const toEnableFields = [];
-                    if (!this.metaData.fields[cm.field1].__computed_id) {
-                        toEnableFields.push(cm.field1);
-                    }
-                    if (!this.metaData.fields[cm.field2].__computed_id) {
-                        toEnableFields.push(cm.field2);
+                if (fieldDef) {
+                    if (fieldDef.__computed_id) {
```

Comment:
Isn't it the same? You can use the shorter form. Anyway this change isn't needed IMO

```js
if (fieldDef?.__computed_id)

Reply:
yes, sure